admin/includes/init_includes/init_sanitize.php

```diff
@@ -238,8 +238,33 @@
 
 $group = array('configuration_key', 'search', 'query_string');
 
-$checks = new queryFactoryResult($db->link);
-// if current page is configuration, the configuration option does not have a val_function and saving the setting, then sanitize configuration_value to array.
+$configs_with_special_characters = array(
+   'BREAD_CRUMBS_SEPARATOR',
+   'BEST_SELLERS_FILLER',
+   'CATEGORIES_SEPARATOR',
+   'CATEGORIES_SEPARATOR_SUBS',
+   'CATEGORIES_COUNT_PREFIX',
+   'CATEGORIES_SUBCATEGORIES_INDENT',
+   'EZPAGES_SEPARATOR_HEADER',
+   'EZPAGES_SEPARATOR_FOOTER',
+   'CURRENCIES_TRANSLATIONS',
+   'STORE_OWNER_EMAIL_ADDRESS',
+   'EMAIL_FROM',
+   'SEND_EXTRA_ORDER_EMAILS_TO',
+   'SEND_EXTRA_CREATE_ACCOUNT_EMAILS_TO_STATUS',

```

Comment:
As mentioned elsewhere, this _TO_STATUS entry should not be in this list.
Just pointing it out here in case someone tries to use this code. NOTE: This code is NOT what Zen Cart has chosen to use to address this issue.